code/game/machinery/accounting.dm

```diff
@@ -29,6 +29,10 @@
 		RegisterSignal(inserted_id, COMSIG_PARENT_QDELETING, .proc/remove_card)
 		var/datum/bank_account/bank_account = new /datum/bank_account(inserted_id.registered_name)
 		inserted_id.registered_account = bank_account
+		if(new_id.trim)
```

Comment:
![image](https://user-images.githubusercontent.com/24975989/128096683-68125693-e2b4-40af-96c9-e67f92c22502.png)

You'll probably want a `if(istype(new_id.trim, /datum/id_trim/job))` and then a cast to a job trim if true so you can get the job from it.